plugins/tiddlywiki/multiwikiserver/modules/store/sql-tiddler-store.js

```diff
@@ -275,6 +276,10 @@ SqlTiddlerStore.prototype.saveBagTiddlerWithAttachment = function(incomingTiddle
 Returns {tiddler_id:,bag_name:}
 */
 SqlTiddlerStore.prototype.saveRecipeTiddler = function(incomingTiddlerFields,recipe_name) {
+	var tiddlerInfo = this.sqlTiddlerDatabase.getRecipeTiddler(incomingTiddlerFields.title,recipe_name);
+	if (incomingTiddlerFields.type?.includes('image') && tiddlerInfo?.attachment_blob) {
+		incomingTiddlerFields['attachment_blob'] = tiddlerInfo.attachment_blob
```

Comment:
I don't think this is right. We should not be setting a field called "attachment_blob" on tiddlers. In the APIs the attachment blob is a separate parameter.